draft-ietf-httpbis-resumable-upload.md

```diff
@@ -522,4 +522,6 @@ However, from experience with the tus project, users are often asking for a way
 
 This document is based on an Internet-Draft specification written by Jiten Mehta, Stefan Matsson, and the authors of this document.
 
+The Initial version of this document was inspired by the [tus v1 protocol](https://tus.io/) which provides a specification and working implementations for a resumable file upload protocol over HTTP. Memebers of its community helped significantly in creating the early Internet-Draft.
```

Comment:
s/memebers/members

Reply:
Also s/Initial/initial

Reply:
Shoot, I should not make PRs at such a late hour!